spring-cloud-dataflow-platform-cloudfoundry/pom.xml

```diff
@@ -26,6 +26,10 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter-config</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
This is a legacy oauth package which we're not using anymore. We can't add it back. Here's a diff for jars after this PR:
```
$ compare-boot-jars /tmp/xxx/difjars//spring-cloud-dataflow-server-2.6.0-SNAPSHOT-1.jar /tmp/xxx/difjars//spring-cloud-dataflow-server-2.6.0-SNAPSHOT-2.jar
65,68c65,68
< BOOT-INF/lib/java-cfenv-boot-1.1.3.RELEASE.jar
< BOOT-INF/lib/java-cfenv-boot-pivotal-scs-1.1.3.RELEASE.jar
< BOOT-INF/lib/java-cfenv-boot-pivotal-sso-1.1.3.RELEASE.jar
< BOOT-INF/lib/java-cfenv-jdbc-1.1.3.RELEASE.jar
---
> BOOT-INF/lib/java-cfenv-boot-2.1.2.RELEASE.jar
> BOOT-INF/lib/java-cfenv-boot-pivotal-scs-2.1.2.RELEASE.jar
> BOOT-INF/lib/java-cfenv-boot-pivotal-sso-2.1.2.RELEASE.jar
> BOOT-INF/lib/java-cfenv-jdbc-2.1.2.RELEASE.jar
70a71
> BOOT-INF/lib/javax.activation-api-1.2.0.jar
74a76,78
> BOOT-INF/lib/jaxb-api-2.3.1.jar
> BOOT-INF/lib/jaxb-core-2.3.0.1.jar
> BOOT-INF/lib/jaxb-impl-2.3.0.1.jar
215a220
> BOOT-INF/lib/spring-security-jwt-1.0.9.RELEASE.jar
216a222
> BOOT-INF/lib/spring-security-oauth2-autoconfigure-2.1.2.RELEASE.jar
```